lesson_10/solid/solid_app/src/main/java/com/codedifferently/lesson10/library/Librarian.java

```diff
@@ -0,0 +1,57 @@
+package com.codedifferently.lesson10.library;
+
+import com.codedifferently.lesson10.library.exceptions.BookCheckedOutException;
+
+/** Represents a librarian of a library. */
+public class Librarian extends Patron {
```

Comment:
Should be integrated with the `Library` class.

Reply:
Not sure if I'm looking at this prematurely, but you still haven't fulfilled the Librarian requirement according to the assignment details.